docs/dev/profiling.md

```diff
@@ -7,177 +7,209 @@ redirect_from:
-│          (0.00s)          ││                           │
-└───────────────────────────┘└─────────────┬─────────────┘
-                             ┌─────────────┴─────────────┐
-                             │         SEQ_SCAN          │
-                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
-                             │          students         │
-                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
-                             │            sid            │
-                             │            name           │
-                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
-                             │ Filters: name>=Ma AND name│
-                             │  <Mb AND name IS NOT NULL │
-                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
-                             │           EC: 1           │
-                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
-                             │             2             │
-                             │          (0.00s)          │
-                             └───────────────────────────┘
-```
+These cumulative metrics can be enabled independently, even if the underlying specific metrics are disabled.
+The table below shows the cumulative metrics and the specific metrics they are calculated from.
+
+| Metric                    | Metric Calculated Cumulatively |
+|---------------------------|--------------------------------|
+| `CPU_TIME`                | `OPERATOR_TIMING`              |
+| `CUMULATIVE_CARDINALITY`  | `OPERATOR_CARDINALITY`         |
+| `CUMULATIVE_ROWS_SCANNED` | `OPERATOR_ROWS_SCANNED`        |
+
+## Detailed Profiling
+
+As explained above, when the `profiling_mode` is set to `detailed`, an extra set of timing metrics are enabled, which are only available at the `QUERY_ROOT` level. These include [`OPTIMIZERS`](#optimizers), [`PLANNER`](#planner), and [`PHYSICAL_PLANNER`](#physical-planner) metrics, which are all measured in milliseconds, and returned as a `double`.
+
+These metrics are automatically enabled, and added to the enabled settings, when the `profiling_mode` is set to `detailed`, however, they can also be toggled individually. 
+
+### Optimizers
+
+At the `QUERY_ROOT` level, there are also metrics that measure the time taken by each [optimizer]({% link docs/internals/overview.md %}#optimizer). These metrics are only available when the specific optimizer is enabled. The available optimizations can be queried using the [`duckdb_optimizers() table function`]({% link docs/sql/meta/duckdb_table_functions.md %}#duckdb_optimizers).
+
+Each optimizer has a corresponding metric that follows the template: `OPTIMIZER_<OPTIMIZER_NAME>`. For example, the `OPTIMIZER_JOIN_ORDER` metric corresponds to the `JOIN_ORDER` optimizer.
+
+Additionally, the following metrics are available to support the optimizer metrics:
+- `ALL_OPTIMIZERS` - Turns on all optimizer metrics, and measures the time taken by the optimizer parent node
```

Comment:
Missing dot. Also for the other bullet lists, not sure if we want them or not haha.